Pass through authentication supports write-back

chadhasbrook opened the pull request:
Added * **Supports password resets for users that use Azure AD Pass-through Authentication**: With password writeback, as long as the user accounts are synchronized into your Azure AD tenant, they are able to manage their on-premises Active Directory passwords from the cloud.

Partner complained this was confusing since we did not explicitly call this out in our documentation.  Please update.

azurecla commented:
Hi __@abaranch__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (abaranch). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

abaranch commented:
Per discussion with @alancameronwills we will need more details. Abandon pull request for now.

chadhasbrook committed:
Pass through authentication supports write-back

Added * **Supports password resets for users that use Azure AD Pass-through Authentication**: With password writeback, as long as the user accounts are synchronized into your Azure AD tenant, they are able to manage their on-premises Active Directory passwords from the cloud.

Partner complained this was confusing since we did not explicitly call this out in our documentation.  Please update.
